tools/clang-display-context/ClangDisplayContext.cpp

```diff
@@ -29,6 +29,7 @@
 #include "llvm/Support/CommandLine.h"
 #include "llvm/Support/Signals.h"
 
+#include <cstdio>
```

Comment:
In LLVM we try to avoid using C APIs, because they are inherently unsafe. I would use a `stringstream` instead of `sprintf`. Alternatively, LLVM `Twine` and `std::to_string` can be used.